kernel/write_uint63.ml

```diff
@@ -1,10 +1,18 @@
-(** Equivalent of rm -f *)
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *   INRIA, CNRS and contributors - Copyright 1999-2018       *)
+(* <O___,, *       (see CREDITS file for the list of authors)           *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
 
+(** Equivalent of rm -f *)
```

Comment:
Indeed now that I see this [it slipped on the native review] you could also remove all this code and just use the primitive `with-stdout-to` to write the file, gaining in both robustness of the generated file layout and avoiding having to mess with files here.